LazyAlchemyLearner.lua

```diff
@@ -142,41 +187,164 @@ local function getItemLinkFromItemId(itemId)
 	return string.format("|H1:item:%d:%d:50:0:0:0:0:0:0:0:0:0:0:0:0:%d:%d:0:0:%d:0|h|h", itemId, 0, ITEMSTYLE_NONE, 0, 10000) 
 end
 
-local function getSolvent(proficiency, start)
+local function getSolvent(proficiency, startingPosition)
```

Comment:
nope, did something wrong here, will fix

Reply:
the green red was confusing me, I'm used to azure devops 2 sided view. Should be good

Reply:
![Screenshot_20241010_161849_Chrome](https://github.com/user-attachments/assets/0dd0a09b-7021-4958-9e12-b7741ead97c5)
Try this, I think this is more like the view you're used to